### PR TITLE
IMP:	- Migrated module sale_order_margin_percent to branch 11.0

### PR DESCRIPTION
**Pull request by OmniaGit**
Migrated module sale_order_margin_percent to branch 11.0

### CODE REVIEW

File: sale_order_margin_percent/__init__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+##############################################################################
```
Comment (nikul-serpentcs) [inline sale_order_margin_percent/__init__.py:1]: @OmniaGit Remove Copyright from `__init__.py` file.

File: sale_order_margin_percent/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#    GNU General Public License for more details.
+#
+##############################################################################
+
+{
+    "name": "Sale Order Margin Percent",
+    "summary": "Show Percent in sale order",
+    "version": "10.0.1.0.0",
+    "category": "Sales",
+    "website": "http://www.pesol.es",
```
Comment (nikul-serpentcs) [inline sale_order_margin_percent/__manifest__.py:17]: @OmniaGit Improve URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10).

File: sale_order_margin_percent/models/sale_order_margin.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+from odoo import models, api, fields
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    percent = fields.Float(
+        string='Percent',
+        compute='_compute_percent')
```
Comment (nikul-serpentcs) [inline sale_order_margin_percent/models/sale_order_margin.py:20]: @OmniaGit IMO add `default=0.0` or `order.percent= 0.0` [here](https://github.com/OCA/sale-workflow/pull/692/files#diff-1f798e0f0bf1d10e29f35ab319481572R24 )

File: sale_order_margin_percent/views/sale_order_margin_percent_view.xml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_sale_order_margin_percent_form" model="ir.ui.view">
+        <field name="name">view.sale.order.margin.percent.form</field>
+        <field name="model">sale.order</field>
+        <field name='inherit_id' ref='sale_margin.sale_margin_sale_order'/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='margin']" position="after">
```
Comment (nikul-serpentcs) [inline sale_order_margin_percent/views/sale_order_margin_percent_view.xml:8]: @OmniaGit Directly use `field` tag

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Review by AaronHForgeFlow — COMMENTED** [review]
Thanks for contributing. Commit history is missing. Please follow the recommended procedure: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

File: sale_order_margin_percent/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    "author": "PESOL, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        'sale',
```
Comment (YogeshMahera-SerpentCS) [inline sale_order_margin_percent/__manifest__.py:23]: @OmniaGit IMHO 

please remove `sale` dependency, `sale_margin` having `sale_management` dependency which have `sale` module!

File: sale_order_margin_percent/models/sale_order_margin.py (hunk 3 above)
Comment (YogeshMahera-SerpentCS) [reply sale_order_margin_percent/models/sale_order_margin.py:20]: @nikul-serpentcs I think float field having `default=0.0`

Comment (pedrobaeza) [reply sale_order_margin_percent/models/sale_order_margin.py:20]: Being a compute, it doesn't need a default. It's automatically reset to that default at the beginning of the compute method.

**Pull request closed without merging**